openpype/tools/settings/settings/widgets.py

```diff
@@ -640,11 +679,27 @@ def __init__(self, parent):
         layout.addWidget(label_widget, 0)
         layout.addWidget(project_list, 1)
 
+        if only_active:
+            inactive_chk = None
+        else:
+            inactive_chk = QtWidgets.QCheckBox(" Show Inactive Projects ")
```

Comment:
Use parenting on widgets for proper style propagation.
```suggestion
            inactive_chk = QtWidgets.QCheckBox(" Show Inactive Projects ", self)
```

Reply:
Hmmm, I tried this but doesn't have any difference.

Reply:
I know but it may differ if we had different styles for checkbox than default.